packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -118,10 +122,29 @@ export interface IOdspCache extends INonPersistentCache {
 }
 
 export class NonPersistentCache implements INonPersistentCache {
+	constructor(readonly snapshotPrefetchResultCache?: SnapshotPrefetchResultCache) {
+		if (this.snapshotPrefetchResultCache === undefined) {
+			this.snapshotPrefetchResultCache = new PromiseCache<
```

Comment:
Curious, can you do here new SnapshotPrefetchResultCache() ?

Reply:
Since now, I am sharing the driver factory as you suggested and cosmin also recommended, we don't need it. So removing.

Reply:
But you can do it what you were asking.